src/table/settings.ini

```diff
@@ -3154,14 +3159,14 @@ interval = 1
 str      = STR_CONFIG_SETTING_DEFAULT_SIGNAL_TYPE
 strhelp  = STR_CONFIG_SETTING_DEFAULT_SIGNAL_TYPE_HELPTEXT
 strval   = STR_CONFIG_SETTING_DEFAULT_SIGNAL_NORMAL
-cat      = SC_BASIC
+cat      = SC_EXPERT
```

Comment:
The "expert" category is intented to contain "weird" settings, which noone wants to touch, but which we did not dare to remove entirely.

Block signals are something for the "advanced" category.
With this change "gui.enable_signal_gui" also belongs into the "advanced" category, since you now need to change it to play with block signals.